org.eclipse.lsp4e/src/org/eclipse/lsp4e/LanguageServersRegistry.java

```diff
@@ -208,6 +215,24 @@ public Class<? extends LanguageServer> getServerInterface() {
 			return super.getServerInterface();
 		}
 
+		@SuppressWarnings("unchecked")
+		@Override
+		public Consumer<GsonBuilder> getConfigureGson() {
```

Comment:
I think you should use the method `createLauncherBuilder` as a template

Reply:
Indeed, I think you can create a helper method that can be reused by both methods, just passing `CONFIGURE_GSON_ATTRIBUTE` or `LAUNCHER_BUILDER_ATTRIBUTE`